src/test/java/br/com/caelum/vraptor/panettone/CompilerTest.java

```diff
@@ -69,6 +70,16 @@ public void testShouldIgnoreExceptions() {
 	}
 
 	@Test
+	public void testShouldContainTemplateFileInExceptions() {
+		io.copy("oi.tone", "<html>Oi<% for a %></html>");
+		List<Exception> exceptions = compiler.compileAll();
+		assertEquals(1, exceptions.size());
+		
+		File template = new File(sources, "oi.tone");
+		assertTrue(exceptions.get(0).getMessage().contains(format("in template[%s]", template.getAbsoluteFile())));
```

Comment:
What do you think about using `assertThat` here? it provide better failure messages.
It'd look like `assertThat(exceptions.get(0).getMessage(), containsString(...))`